src/main/resources/templates/calendar1.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:th="http://www.thymeleaf.org">
+<head>
+	<meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+	<title>National Cybersecurity Laboratories - Calendar</title>
+	<object th:include="fragments/header :: header" th:remove="tag" />
+</head>
+	<body class="mybg loginbackground">
+	<div th:replace="fragments/dashboard_navigation :: dashboardnav">&nbsp;</div>
+    <p>&nbsp;</p>
+	<p>&nbsp;</p>
+
+
+	<div align="center">
+	<iframe src="https://calendar.google.com/calendar/embed?height=600&amp;wkst=1&amp;bgcolor=%23FFFFFF&amp;src=ljff3k0eksf9ek3c1upgag5p6k%40group.calendar.google.com&amp;color=%230F4B38&amp;ctz=Asia%2FSingapore" style="border-width:0" width="800" height="600" frameborder="0" scrolling="no"></iframe>
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this deprecated "frameborder" attribute. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3AS1827)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this deprecated "scrolling" attribute. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3AS1827)